.devcontainer/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:16-buster-slim
+# install base development tools
+RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync chromium parallel python3
```

It looks like I need the canvas dependencies on ARM so I can build node canvas. 

```suggestion
RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync chromium parallel python3 libpixman-1-dev libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev
```

---

ok, just trying it here.

---

Image build is fin as well here.

I am still not happy about the jest plugin giving an error.
But I know too little abou typescript to judge it's usefulness, compared to running "yarn test".

Maybe the jest plugin is not needed. 

But I leave that to you to decide.